cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/index.tsx

```diff
@@ -38,18 +43,65 @@ export default class DeployedPipelineView extends React.PureComponent {
   }
 
   public render() {
-    return (
-      <Provider store={Store}>
-        <div className="pipeline-deployed-view pipeline-list-content">
-          <div className="deployed-header">
-            <PipelineCount />
-            <SearchBox />
-            <Pagination />
-          </div>
-
-          <PipelineTable />
-        </div>
-      </Provider>
-    );
+    return <DeployedPipelinesView />;
   }
 }
+
+const currentNamespace = getCurrentNamespace();
+
+const DeployedPipelinesView = () => (
+  <Query
+    query={gql`
+      {
+        applications(namespace: "${currentNamespace}") {
```

Comment:
you need to pass the `artifactName` here, right?

Reply:
Not necessarily. See my comment above. The schema is passing a default value to `artifactName` if one is not explicit defined in the query.

Reply:
Fixed as per the discussion above